server/davidia/demos/simple.py

```diff
@@ -102,6 +102,24 @@ def heatmap_demo(p):
     )
 
 
+def large_heatmap_demo(p):
```

Comment:
Not really a "large" heatmap. I would add an optional keyword argument to `heatmap_demo` like `high=False`...

Reply:
Added keyword arg